analyzers/tests/SonarAnalyzer.UnitTest/TestCases/NativeMethodsShouldBeWrapped.CSharp11.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Runtime.InteropServices;
+
+public interface IInterface
+{
+    [DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
+    static extern virtual bool RemoveDirectory1(string name); // Noncompliant
+
+    [DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
+    public static extern virtual bool RemoveDirectory2(string name);  // Noncompliant
+}
```

Comment:
Copy of https://github.com/SonarSource/sonar-dotnet/blob/c18ed76dbbc997007e41be9e9e49df3aa68ab233/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/PInvokesShouldNotBeVisible.CSharp11.cs#L1-L10